rvsol/test/RISCV.t.sol

```diff
@@ -2357,6 +2357,83 @@ contract RISCV_Test is CommonTest {
         assertEq(postState, outputState(expect), "unexpected post state");
     }
 
+    /* Revert cases */
+
+    function test_unknown_instruction() public {
+        uint32 insn = encodeRType(0xff, 0, 0, 0, 0, 0); // 0xff is unknown instruction opcode
+        (State memory state, bytes memory proof) = constructRISCVState(0, insn);
+        bytes memory encodedState = encodeState(state);
+
+        vm.expectRevert(hex"00000000000000000000000000000000000000000000000000000000f001c0de");
```

Comment:
Can we modularize these error codes? Like https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/src/cannon/libraries/CannonErrors.sol

Reply:
I've considered that, but it's not easy because RISCV.sol is a yul code. so I think the current way is most straightforward code. what do you think?

Reply:
```solidity
contract Foo {
    error MyError();
    bytes4 myErrorSelector = MyError.selector;
    function step() public {
        assembly {
                mstore(0, sload(myErrorSelector.slot))
                revert(0, 4)
        }
    }
}
```

We can declare error and selectors for errors at `RISCV.sol`.

Reply:
That looks good, but it needs much diff. let me do it as another PR.